Debian policy.
This patch updates default plugin path and config path.
It also updates the pcsx2 game db path and shaders data path.
AppInit.cpp:93 could be probably updated
Index: pcsx2-0.9.8/pcsx2/gui/AppConfig.cpp
===================================================================
--- pcsx2-0.9.8.orig/pcsx2/gui/AppConfig.cpp	2011-04-10 15:51:53.000000000 +0200
+++ pcsx2-0.9.8/pcsx2/gui/AppConfig.cpp	2011-04-10 16:10:07.000000000 +0200
@@ -169,7 +169,8 @@
 
 	wxDirName GetPlugins()
 	{
-		return AppRoot() + Base::Plugins();
+		// return AppRoot() + Base::Plugins();
+		return wxDirName( L"/usr/lib/games/pcsx2" ) + Base::Plugins();
 	}
 
 	wxDirName GetThemes()
Index: pcsx2-0.9.8/plugins/zzogl-pg/opengl/ZZoglCreate.cpp
===================================================================
--- pcsx2-0.9.8.orig/plugins/zzogl-pg/opengl/ZZoglCreate.cpp	2011-04-10 15:51:53.000000000 +0200
+++ pcsx2-0.9.8/plugins/zzogl-pg/opengl/ZZoglCreate.cpp	2011-04-10 16:11:08.000000000 +0200
@@ -299,8 +299,11 @@
 
 		if (fres == NULL)
 		{
-			ZZLog::Error_Log("Cannot find ps2hw.dat in working directory. Exiting.");
-			return false;
+            fres = fopen("/usr/share/games/pcsx2/shaders/ps2hw.dat", "rb");
+            if (fres == NULL) {
+                ZZLog::Error_Log("Cannot find ps2hw.dat in working directory. Exiting.");
+                return false;
+            }
 		}
 	}
 
Index: pcsx2-0.9.8/pcsx2/gui/AppGameDatabase.h
===================================================================
--- pcsx2-0.9.8.orig/pcsx2/gui/AppGameDatabase.h	2011-04-10 15:51:53.000000000 +0200
+++ pcsx2-0.9.8/pcsx2/gui/AppGameDatabase.h	2011-04-10 16:11:42.000000000 +0200
@@ -51,8 +51,8 @@
 		Console.WriteLn( "(GameDB) Unloading..." );
 	}
 
-	AppGameDatabase& LoadFromFile(const wxString& file = L"GameIndex.dbf", const wxString& key = L"Serial" );
-	void SaveToFile(const wxString& file = L"GameIndex.dbf");
+	AppGameDatabase& LoadFromFile(const wxString& file = L"/var/games/pcsx2/GameIndex.dbf", const wxString& key = L"Serial" );
+	void SaveToFile(const wxString& file = L"/var/games/pcsx2/GameIndex.dbf");
 };
 
 static wxString compatToStringWX(int compat) {
